pom.xml

```diff
@@ -107,6 +107,22 @@
             <artifactId>testng</artifactId>
             <version>7.4.0</version>
         </dependency>
+        <dependency>
+            <groupId>io.cucumber</groupId>
+            <artifactId>cucumber-java</artifactId>
+            <version>4.8.1</version>
+        </dependency>
+        <dependency>
+            <groupId>io.cucumber</groupId>
+            <artifactId>cucumber-testng</artifactId>
+            <version>4.8.1</version>
+        </dependency>
+        <dependency>
+            <groupId>io.qameta.allure</groupId>
```

Comment:
we are using yandex allure, not qameta

Reply:
Deleted, thank you.
Please see in the following commit.